src/Form.php

```diff
@@ -0,0 +1,1202 @@
+<?php
+
+/**
+ * Class Form
+ *
+ * @package TrustedLogin\Client
+ *
+ * @copyright 2021 Katz Web Services, Inc.
+ */
+
+namespace TrustedLogin;
+
+// Exit if accessed directly
+if (!defined('ABSPATH')) {
+	exit;
+}
+
+use \WP_User;
+
+
+/**
+ * Creates the TrustedLogin support user form.
+ * 	- Makes the HTML
+ *  - Manages assets
+ * Does not
+ * 	- Handle the form submission ./Ajax.php
+ *  - Setup the menu ./Admin.php
+ */
+final class Form
+{
+
+	/**
+	 * URL pointing to the "About TrustedLogin" page, shown below the Grant Access dialog
+	 */
+	const ABOUT_TL_URL = 'https://www.trustedlogin.com/about/easy-and-safe/';
+
+	const ABOUT_LIVE_ACCESS_URL = 'https://www.trustedlogin.com/about/live-access/';
+
+	/**
+	 * @var Config
+	 */
+	private $config;
+
+	/**
+	 * @var SiteAccess $site_access
+	 */
+	private $site_access;
+
+	/**
+	 * @var SupportUser $support_user
+	 */
+	private $support_user;
+
+	/**
+	 * @var null|Logging $logging
+	 */
+	private $logging;
+
+	/**
+	 * Admin constructor.
+	 *
+	 * @param Config $config
+	 */
+	public function __construct(Config $config, Logging $logging,SupportUser $support_user, SiteAccess $site_access)
```

Comment:
Formatting isn't consistent with other files; I can fix this though.

Reply:
9d5b85614e674bb6c6e2d135beea81a93ff45f0b